app/main.py

```diff
@@ -1 +1,38 @@
-#  write your code here
+class Animal:
+    def __init__(self, name, appetite, is_hungry=True):
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self):
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self):
+        if self.is_hungry:
+            print(f"Eating {self.appetite} food points...")
+            self.is_hungry = False
+            return self.appetite
+        else:
```

Comment:
You don't have to use the `else` block if you have a `return` statement in the `if` block. Just `return 0`